.github/workflows/codacy.yml

```
@@ -11,6 +11,17 @@ jobs:
     codacy:
         name: Codacy
         runs-on: ubuntu-latest
+
+        services:
+            mysql:
+                image: mysql:8.0
+                env:
+                    MYSQL_ROOT_PASSWORD: root
+                    MYSQL_DATABASE: pancake
+                ports:
```

actionlint

could not parse as YAML: yaml: line 21: did not find expected '-' indicator

(syntax-check)